src/index.js

```diff
@@ -170,18 +174,21 @@ const oneAppDevCdnFactory = ({
       const remoteModuleBaseUrlOrigin = new URL(knownRemoteModuleBaseUrl).origin;
       got(`${remoteModuleBaseUrlOrigin}/${req.path}`, {
         headers: { connection: 'keep-alive' },
-        agent: new ProxyAgent(),
+        agent: {
+          https: new ProxyAgent(),
+          http: new ProxyAgent(),
+        },
       })
-        .then((remoteModuleResponse) => remoteModuleResponse.body)
-        .then(
-          (remoteModuleResponse) => res
-            .status(200)
-            .type(path.extname(req.path))
-            .send(remoteModuleResponse),
-          (err) => res
-            .status(500)
-            .send(err.message)
-        );
+        .then((remoteModuleResponse) => res
+          .status(res.statusCode)
+          .type(path.extname(req.path))
+          .send(remoteModuleResponse.body))
+        .catch((err) => {
+          const status = err.code === 'ERR_NON_2XX_3XX_RESPONSE' ? err.response.statusCode : 500;
```

Comment:
very tempted to strip out got and replace with node fetch

Reply:
+1 to that.